android/app/src/main/java/org/fossasia/openevent/fragments/SpeakersListFragment.java

```diff
@@ -118,6 +120,7 @@ public void onRefresh() {
                     public void accept(@NonNull List<Speaker> speakers) throws Exception {
                         mSpeakers.clear();
                         mSpeakers.addAll(speakers);
+                        array = DbSingleton.getInstance().getCount(mSpeakers).get(0);
```

Comment:
@iamareebjamal please check the method in DbSingleton.java
i am not querying the list, i am using the speaker list to get the count instead of over here. i think this should be fine.